org.upscayl.app.yml

```diff
@@ -0,0 +1,70 @@
+id: org.upscayl.app
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: upscayl-run
+separate-locales: false
+
+finish-args:
+  - --allow=devel
+  - --allow=multiarch
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+
+modules:
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2022.04
+  - name: upscayl
+    buildsystem: simple
+    cleanup:
+      - upscayl-2.0.1-linux
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: >-
+          https://github.com/upscayl/upscayl/releases/download/v2.0.1/upscayl-2.0.1-linux.zip
+        sha256: 96883ae1bb9a3a5c0972ab7e402c5132e40c4fe0ffaeec8753524df16b03de51
+      - type: file
+        path: org.upscayl.app.metainfo.xml
+      - type: file
+        path: icon.png
+      - type: file
+        path: org.upscayl.app.desktop
+      - type: script
+        dest-filename: upscayl-run
+        commands:
+          - zypak-wrapper /app/upscayl/upscayl
+    build-commands:
+      - install -d /app/upscayl
+      - unzip upscayl-2.0.1-linux -d /app/upscayl
+      - install upscayl-run /app/bin/
+      - install -d /app/share/icons/hicolor/512x512/apps/
+      - >-
+        install icon.png
+        /app/share/icons/hicolor/512x512/apps/org.upscayl.app.png
+      - install -d /app/share/applications
+      - >-
+        install -Dp -m644 org.upscayl.app.metainfo.xml
+        /app/share/metainfo/org.upscayl.app.metainfo.xml
+      - >-
+        install -m644 org.upscayl.app.desktop
+        /app/share/applications/org.upscayl.app.desktop
+      - >-
+        desktop-file-edit --set-key=Exec --set-value='upscayl-run %U'
+        --set-key=Icon --set-value='org.upscayl.app'
+        --set-key=Comment --set-value='Upscale Images' --set-key=Name
+        --set-value='Upscayl' --set-key=StartupNotify --set-value='false'
+        --set-key=Categories
+        --set-value='ImageProcessing;RasterGraphics;Graphics;' --set-key=Type
+        --set-value='Application'
```

Comment:
At this point I would consider modifying the desktop file in your git repo directly and making a new release. It basically rewrites most of it.